src/main.rs

```diff
@@ -1383,6 +1383,7 @@ fn validate_signed_delegate_action(
     let da_receiver_id = &signed_delegate_action.delegate_action.receiver_id;
 
     // if we are not using whitelisted contracts or senders, then no validation needed
+    // TODO: Is this true? What about validating method name? Or Pay With FT?
```

Comment:
There's no supported config that validates method name and allows `use_whitelisted_contracts = false` and `use_whitelisted_senders = false`. `pay_with_ft.toml` should use whitelisted contracts since the ft contract address would need to be whitelisted.

Reply:
Is it not supported because we declare it so? Or for some technical reason?

Reply:
>All of those configs either use whitelisted contracts or senders  

Is this just by coincidence? Like it nets out such that it's implicitly disallowed to have non-whitelisted contracts/senders _and_ perform method name checks?

Reply:
You could hypothetically create such a case where inspect the content of the method without checking the contract or sender, but there are no use cases that correspond to that scenario.